ddp-workspace/projects/ddp-pancan/src/app/app.module.ts

```diff
@@ -48,6 +53,11 @@ toolkitConfig.doneUrl = AppRoutes.AgeUpThankYouProxy;
 toolkitConfig.errorUrl = AppRoutes.Error;
 toolkitConfig.phone = '651-287-1608';
 toolkitConfig.infoEmail = 'info@joincountmein.org';
+toolkitConfig.twitterAccountId = 'count_me_in';
```

Comment:
(no need to change anything) I see that these changes are not related to the new page and think It might be good in the future to create separate PRs for the changes which are not crossed or related to each other and might be merged independently. It helps to make review faster and decrese the PR size.

Reply:
Agree that it is worth separating PR's to be smaller.
Let's try it next.